server/utils/nunjucksSetup.ts

```diff
@@ -85,23 +85,30 @@ export function registerNunjucks(app?: express.Express): Environment {
   })
 
   njkEnv.addFilter('displayAge', (dateOfBirth: string) => {
-    let ageString
-
     const today = new Date()
     const dob = new Date(dateOfBirth)
-    const years = today.getFullYear() - dob.getFullYear()
-    if (years > 1) {
+
```

Comment:
It just occurred to me, is there a reason you're not using the built-in `date-fn` library we use on the project. I really have to apologise for not spotting that!!!

Reply:
https://date-fns.org/docs/Getting-Started

Reply:
mentioned in the description. Tried it with date fns but that also leads you to have to make these work-arounds with particular dates (It struggled with months) 
That's what led me to make this manual calculation instead of part Date FNS, part our own

Reply:
Fair enough!